symplyphysics/laws/optics/optical_distance_from_geometric_distance_and_refractive_index.py

```diff
@@ -0,0 +1,33 @@
+from sympy import (Eq, solve)
+from symplyphysics import (units, Quantity, Symbol, print_expression, dimensionless, validate_input,
+    validate_output)
+
+# Description
+## Optical distance L is a geometric distance l in optical environment with refractive index n
+
+# Law: L = n * l
```

Comment:
We already have this: https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/laws/waves/optical_path_length_from_geometric_path_length_and_refractive_index.py